From ed7237214b9114082c99cd2a6d4d4529981e9808 Mon Sep 17 00:00:00 2001
From: Hyun Kwon <hyun.kwon@xilinx.com>
Date: Thu, 3 Sep 2020 12:44:06 -0700
Subject: [PATCH 1563/1851] v4l: xilinx: scaler: Set the pm ops

commit f24118fe958e7103821fde7589e1ef1c0a40d2d9 from
https://github.com/Xilinx/linux-xlnx.git

drivers/media/platform/xilinx/xilinx-scaler.c:688:26: warning: 'xscaler_pm_ops' defined but not used [-Wunused-const-variable=]
  688 | static SIMPLE_DEV_PM_OPS(xscaler_pm_ops, xscaler_pm_suspend, xscaler_pm_resume);
      |                          ^~~~~~~~~~~~~~
include/linux/pm.h:354:25: note: in definition of macro 'SIMPLE_DEV_PM_OPS'
  354 | const struct dev_pm_ops name = { \
      |                         ^~~~

Signed-off-by: Hyun Kwon <hyun.kwon@xilinx.com>
Reviewed-by: Vishal Sagar <vishal.sagar@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-scaler.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/media/platform/xilinx/xilinx-scaler.c b/drivers/media/platform/xilinx/xilinx-scaler.c
index bb0d52627a50..9cb22a5a8966 100644
--- a/drivers/media/platform/xilinx/xilinx-scaler.c
+++ b/drivers/media/platform/xilinx/xilinx-scaler.c
@@ -696,6 +696,7 @@ MODULE_DEVICE_TABLE(of, xscaler_of_id_table);
 static struct platform_driver xscaler_driver = {
 	.driver			= {
 		.name		= "xilinx-scaler",
+		.pm		= &xscaler_pm_ops,
 		.of_match_table	= xscaler_of_id_table,
 	},
 	.probe			= xscaler_probe,
-- 
2.31.1

